test/e2e/user_management_test.go

```diff
@@ -125,6 +128,9 @@ func (s *userManagementTestSuite) TestUserDeactivation() {
 			RequireConditions(ConditionSet(Default(), ApprovedByAdmin())...).
 			Execute().Resources()
 
+		// TODO remove once UserTier migration is completed
+		s.promoteToDefaultUserTier(hostAwait.Client, mur2)
+
```

Comment:
the same here and also in some other cases

Reply:
Removed this and other cases in a6cd3440370feefacd130d319af1cd0fc90c9990